### ISSUE TITLE
Module 14: Deploying apps with Chef on Azure

### ISSUE THREAD

**Issue by devops-app** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#56]
# Module: 14
#### Step: 01

Description of issue
![image](https://user-images.githubusercontent.com/65710369/111858293-c6d42100-8972-11eb-95c8-3703a3291ceb.png)

https://gallery.azure.com is deprecated.

Repro steps:

1.
1.
1.

**Comment by TesterTesterson1004** [comment]
Any thoughts from the course owner on directions a fix for this issue could take?

I'm unable to find any solutions, and although I did try using a manual setup of the generic Chef Automate VM I found in the Azure marketplace, which does create such a VM...the SSH/password method the instructions reference does not seem to be an option. I instead used puttygen to create public/private keys and connected using that, but for some reason the next util needed to id the VM fails:

![image](https://user-images.githubusercontent.com/49700217/114204570-31430600-990e-11eb-89e7-63eb2deb902d.png)

Unfortunately, not being a chef/linux expert it is unclear to me how to proceed - attempting to sudo install that cmd failed, quick googling seems to indicate that error occurs if the chef client is installed, rather then chef control server.

Essentially, this entire lab is kaput with this issue.
